payment/src/main/java/org/killbill/billing/payment/api/DefaultPaymentApi.java

```diff
@@ -722,31 +722,31 @@ public Payment createChargebackReversal(final Account account, final UUID paymen
     }
 
     @Override
-    public List<Payment> getAccountPayments(final UUID accountId, final boolean withPluginInfo, final Iterable<PluginProperty> properties, final TenantContext tenantContext) throws PaymentApiException {
+    public List<Payment> getAccountPayments(final UUID accountId, final boolean withPluginInfo, final boolean withAttempts, final Iterable<PluginProperty> properties, final TenantContext tenantContext) throws PaymentApiException {
         return paymentProcessor.getAccountPayments(accountId, withPluginInfo, tenantContext, internalCallContextFactory.createInternalTenantContext(accountId, tenantContext));
```

Comment:
The new flag `withAttempts` is not passed (used) the  `paymentProcessor#getAccountPayments`